.gitignore

```diff
@@ -64,4 +64,7 @@ docs/_build/
 target/
 
 # IPython Notebooks
-.ipynb_checkpoints/*
+.ipynb_checkpoints
```

Comment:
Interesting - it was working for me. Can you please edit the .gitignore to also include `.ipynb_checkpoints/*`? Just to make sure it works on both of our ends.